Dockerfile

```diff
@@ -12,21 +12,30 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 	liblapack-dev \
 	libopenblas-dev \
 	libssl-dev xz-utils \
-	libclang-3.9-dev clang libxml2 zlib1g-dev \
+	libclang-6.0-dev clang libxml2 zlib1g-dev \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.gold" 20 \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.bfd" 10
 
-ENV DLANG_VERSION "dmd-nightly"
-ENV DLANG_EXEC "dmd"
+ARG DLANG_VERSION=dmd
+ARG DLANG_EXEC=dmd
+ENV DLANG_VERSION=$DLANG_VERSION
+ENV DLANG_EXEC=$DLANG_EXEC
 
 # Download and run the install script
 RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh
 RUN bash /tmp/install.sh -p /dlang install ${DLANG_VERSION}
 
+COPY ./har /tmp/har/src
+SHELL ["/bin/bash", "-c"]
+RUN source /dlang/$(ls -tr /dlang | tail -n1)/activate; \
+  echo $PATH && \
+  $DMD -of=/tmp/har/src/har -g -debug /tmp/har/src/harmain.d /tmp/har/src/archive/har.d
```

Comment:
@wilzbach @Geod24  It's ugly but I didn't find a better way. Do you guys have any suggestions?